README.md

```diff
@@ -33,4 +35,4 @@ GrowersBrains is an app designed to solve the problems that growers are globally
 - Clone or fork the repository
 - Install dependencies with NPM `npm i`
 - Create a file named `.env.local` at the root of the project, and add this line to it: `PUBLIC_URL=/`
-- Start the project locally `npm start`
\ No newline at end of file
+- Start the project locally `npm start`
```

Comment:
I think we should additionally add `yarn start` and add `yarn` to the install line. In case anyone has a preference. I slightly prefer to use yarn :)